### DISCUSSION TITLE
第2节课笔记(3班)

### DISCUSSION THREAD

**Discussion by matrixgame2018** [InternLM/Tutorial#100] [笔记]
第2节课笔记(3班)

**Comment by zzmbhm** [comment]
https://z07ck55wvo.feishu.cn/docx/LjqkdMeJ8oBW3yxsKI0cq9ADnie

https://zhuanlan.zhihu.com/p/675985021 实战1：使用InternStudio完成趣味Demo
https://zhuanlan.zhihu.com/p/676039643 实战2：使用InternStudio体验Lagent
https://zhuanlan.zhihu.com/p/675882989 实战3 在WSL2 ubuntu 22.04体验浦语·灵笔图文理解创作
https://zhuanlan.zhihu.com/p/676096599 实战4 在WSL2 ubuntu 22.04体验趣味Demo
介绍了本次课程的内容，包括大模型的区别演示、通用环境配置和课后作业。他还展示了两个demo：智能对话和legend的工具调用。最后分享了如何在服务器上搭建环境和运行模型的步骤。
亮点
•大模型介绍及特点
•智能对话demo
•Legend的工具调用demo
•服务器上搭建环境的步骤
•模型的下载和运行

**Comment by SchweitzerGAO** [comment]
https://schweitzergao.github.io/2024/01/06/mmtutorial-2-note/

**Comment by GoooHi** [comment]
https://zhuanlan.zhihu.com/p/676353574

**Comment by zzmbhm** [comment]
https://docs.qq.com/doc/DZmpHTVNubU9XeHBr

**Comment by wm901115nwpu** [comment]
https://zhuanlan.zhihu.com/p/676426118

**Comment by GoldWaterFall** [comment]
https://www.yuque.com/woshijinshui/wwi5m1/cuh522x3mon3ebt7?singleDoc# 《浦语大模型趣味 Demo 笔记》

**Comment by Gty-2002** [comment]
https://npumgk6rgc.feishu.cn/docx/RUgydCJz4oxn0VxPBpKcbWcCnKd?from=from_copylink （qwqq感觉开始变难了orz

**Comment by Tallisgo** [comment]
http://t.csdnimg.cn/oYMaX

**Comment by xiexiaoshinick** [comment]
<style> td {white-space:nowrap;border:1px solid #dee0e3;font-size:10pt;font-style:normal;font-weight:normal;vertical-align:middle;word-break:normal;word-wrap:normal;}</style><byte-sheet-html-origin data-id="1704623962065" data-version="4" data-is-embed="false" data-grid-line-hidden="false" data-importRangeRawData-spreadSource="https://pukas2rwon.feishu.cn/sheets/X4EgsrqqShEoBwtKA6BczTVLnsd" data-importRangeRawData-range="&#39;3班表格&#39;!H38:H38">
笔记2-轻松玩转书生·浦语大模型趣味Demo和作业
--


</byte-sheet-html-origin>[笔记2-轻松玩转书生·浦语大模型趣味Demo和作业](https://zhuanlan.zhihu.com/p/676521324)

**Comment by SuperD0122** [comment]
https://blog.csdn.net/m0_43404473/article/details/135443505?spm=1001.2014.3001.5501 笔记在前半部分

**Comment by strawberrylazy** [comment]
https://github.com/strawberrylazy/InterLM_Lesson/blob/main/lesson_2/%E7%AC%AC%E4%BA%8C%E8%8A%82.md

**Comment by hydexi0122** [comment]
https://zhuanlan.zhihu.com/p/676562047

**Comment by Sampson425** [comment]
https://mp.csdn.net/mp_blog/creation/editor/135451357

**Comment by G-Mie** [comment]
https://blog.csdn.net/YYYYbhjbhkv_/article/details/135469880

**Comment by Zhihao-z** [comment]
https://blog.csdn.net/m0_46870112/article/details/135588993?spm=1001.2014.3001.5501
